config/initializers/authentication.rb

```diff
@@ -0,0 +1,21 @@
+# Check for authentication method and copy data over if necessary (ENV variable
+# to skip if necessary)
+unless ENV['SKIP_AUTH_INIT']
```

Comment:
Okay, so we will need to explicitly be setting this env variable then when running rake tasks.